### DISCUSSION TITLE
Can Enzyme be used in TACO?

### DISCUSSION THREAD

**Discussion by FDInSky** [EnzymeAD/Enzyme#1078] [Q&A]
https://github.com/tensor-compiler/taco 
I want to make taco's tensor computation with auto-diff. 
Is it possible?

**Comment by wsmoses** [comment]
I'm not an expert on TACO (though have spoken with Fred a couple of times about it), but from my understanding it should definitely be possible.

To my knowledge TACO's dense/sparse codes don't have any code that wouldn't be amenable to static analysis given standard type information.
